### PR TITLE
gdb_main: don't print debug for vMustReplyEmpty packets

### PR DESCRIPTION
**Pull request by perigoso**
## Detailed description

vMustReplyEmpty is a known and correctly handled packet, and thus we should not be printing a warning for it

## Your checklist for this pull request

* [x] I've read the [Code of Conduct](https://github.com/blackmagic-debug/blackmagic/blob/main/CODE_OF_CONDUCT.md)
* [x] I've read the [guidelines for contributing](https://github.com/blackmagic-debug/blackmagic/blob/main/CONTRIBUTING.md) to this repository
* [x] It builds for hardware native (`make PROBE_HOST=native`)
* [x] It builds as BMDA (`make PROBE_HOST=hosted`)
* [x] I've tested it to the best of my ability
* [x] My commit messages provide a useful short description of what the commits do

### CODE REVIEW

File: src/gdb_main.c [hunk 1]
```diff
@@ -668,7 +668,10 @@ static void handle_v_packet(char *packet, const size_t plen)
 			gdb_putpacketz("OK");
 
 	} else {
-		DEBUG_GDB("*** Unsupported packet: %s\n", packet);
+		/* The vMustReplyEmpty is used as a feature test to check how gdbserver handles unknown packets*/
+		/* print only if actually unknown */
+		if (strcmp(packet, "vMustReplyEmpty"))
```
Comment (dragonmux) [inline src/gdb_main.c:673]: Would suggest adding the `!= 0` on the end to make it more explicit that we're testing for non-zero here.. but this works.

**Review by dragonmux — APPROVED** [review]
LGTM, we'll merge unless you wish to deal with the one bit of feedback we've got

File: src/gdb_main.c (hunk 1 above)
Comment (perigoso) [reply src/gdb_main.c:673]: Sure, that's a good thought

**Review by dragonmux — APPROVED** [review]
LGTM, Merging.
